ddl/ddl_db_test.go

```diff
@@ -454,6 +454,27 @@ func (s *testDBSuite) testAlterLock(c *C) {
 	s.mustExec(c, "alter table t_indx_lock add index (c1, c2), lock=none")
 }
 
+func (s *testDBSuite) TestAddMultiColumnsIndex(c *C) {
+	s.tk = testkit.NewTestKit(c, s.store)
+	s.tk.MustExec("use " + s.schemaName)
+
+	s.tk.MustExec("drop database if exists tidb;")
+	s.tk.MustExec("create database tidb;")
+	s.tk.MustExec("use tidb;")
+	s.tk.MustExec("create table tidb.test (a int auto_increment primary key, b int);")
+	s.tk.MustExec("insert tidb.test values (1, 1);")
+	s.tk.MustExec("begin;")
```

Comment:
This test is not easy to understand. Please try to simplify it. Some command like 'begin' or 'commit' is not needed,  `insert` or `insert into` appears everywhere.